Dockerfile

```diff
@@ -16,12 +16,16 @@ RUN \
   apt-get -y --force-yes --no-install-recommends install libfontconfig curl ca-certificates git jq && \
   curl https://grafanarel.s3.amazonaws.com/builds/grafana_${GRAFANA_VERSION}_amd64.deb > /tmp/grafana.deb && \
   dpkg -i /tmp/grafana.deb && \
-  rm -f /tmp/grafana.deb && \
-  curl -L https://github.com/tianon/gosu/releases/download/1.10/gosu-amd64 > /usr/sbin/gosu && \
-  chmod +x /usr/sbin/gosu && \
-  for plugin in $(curl -s https://grafana.net/api/plugins?orderBy=name | jq '.items[] | select(.internal=='false') | .slug' | tr -d '"'); do grafana-cli --pluginsDir "${GF_PLUGIN_DIR}" plugins install $plugin; done && \
+  rm -f /tmp/grafana.deb
+
+RUN  curl -L https://github.com/tianon/gosu/releases/download/1.10/gosu-amd64 > /usr/sbin/gosu && \
```

Comment:
Acceptable, I can rewrite it as 1 RUN, I made it 3 RUNs so it's easier to debug the built operation..

Are you familiar with docker-squash?
https://pypi.python.org/pypi/docker-squash

It essentially flattens undesirable layers:
Squashing helps with organizing images in logical layers. Instead of having an image with multiple (in almost all cases) unnecessary layers - we can control the structure of the image.

Implementing a squash after image build would nullify your valid argument...
and make it easier to make changes to Dockerfile if need be..
(Possible through a Makefile, which runs a script which runs 'Docker build')